src/components/Categories/MainCategoryTile.vue

```diff
@@ -128,29 +127,48 @@ export default {
 			}
 			return true;
 		},
-	}
+		altText() {
+			return `Kiva loans to ${this.categoryName}`;
+		},
+		cleanURL() {
+			return this.categoryUrl.substring(this.categoryUrl.lastIndexOf('/') + 1);
+		},
+		chooseSmallImage() {
+			return this.retinaImageExists
+				? [['small', this.image], ['small retina', this.retinaImage]]
+				: [['small', this.image]];
+		}
+	},
+	methods: {
+		chooseImage(tileSize) {
+			if (tileSize === 'large') {
+				return this.retinaImageExists
+					? [['small', this.retinaImage], ['small retina', this.retinaImage]]
+					: [['small', this.image]];
+			}
+			if (tileSize === 'medium') {
+				return this.retinaImageExists
+					? [['small', this.retinaImage], ['small retina', this.retinaImage],
+						['large', this.image], ['large retina', this.retinaImage]]
+					: [['small', this.image]];
+			}
+			return [['small', this.image]];
```

Comment:
You can make this default return be the case for small tiles. Just make sure to add a comment like: 
// Default case or tileSize === 'small'